### ISSUE TITLE
bartenders catering to frenchies cannot possibly make money if the white flag waver orders champagne

### ISSUE THREAD

**Issue by Paxilmaniac** [tgstation/tgstation#57212]
## Let me explain:
~~The price of champagne at round start is about 600 credits, and you get 100 units of the drink. Patrons want 20 units minimum, so you get five glasses out of the bottle. 600 / 5 = 120, meaning you would need to be paid 120 credits to make all of your money back. Let me remind you that you get paid 35. 
Ok, let's say the economy has changed, and deflation has occurred. This makes the price of champagne bottles about 300 credits, the average for most rounds.  Once again however, 300 / 5 = 60, making you lose 25 credits per glass.
Surely this madness ends right? Surely there is some ideal situation where you have spent the entire round manipulating the economy to remove money from circulation and deflate the value of credits as much as possible, and you have achieved the lowest price for a bottle that I have seen, about 200 credits each. You get where this is going right? 200 / 5 = 40, meaning you lose 5 credits each glass you serve.~~

Apparently you only need ten units to complete orders, maybe less. Still, the price of some of these drinks is a bit quirky
## In conclusion:
Fuck frenchies
